src/main/java/com/frcteam3636/frc2024/subsystems/climber/ClimberIO.kt

```diff
@@ -26,9 +29,42 @@ interface ClimberIO {
     fun moveClimber(speed: Double)
 }
 
+class ClimberIOSim : ClimberIO {
+    companion object {
```

Comment:
Our convention is to name these objects `Constants` and put them at the bottom of the enclosing class.